src/main/kotlin/com/bitjourney/plantuml/Main.kt

```diff
@@ -65,7 +63,7 @@ class Main {
 
     val logger = LoggerFactory.getLogger(Main::class.java)
 
-    val transcoder = TranscoderImpl(AsciiEncoder(), CompressionZlib())
```

Comment:
`TranscoderImpl` is no longer provided as the same API before, but `TranscoderSmart` seems smarter.